.github/workflows/check-api.yml

```diff
@@ -41,17 +41,26 @@ jobs:
     steps:
       - name: Echo workflow run information
         run: |
-          echo "Triggering event name: \"${{ github.event.name }}\", \
+          echo "Triggering event name: \"${{ github.event_name }}\", \
           APIs to check: \"${{ github.event.inputs.environment }}\""
```

Comment:
Let's get rid of the `\"` from the messages here, it clutters up the log message in the Action run, and we already have the `:` separator to distinguish the message label from the value.